install/0001_00_cluster-version-operator_03_service.yaml

```diff
@@ -1,16 +1,19 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: cluster-version-operator
+  name: metrics
   namespace: openshift-cluster-version
   labels:
     k8s-app: cluster-version-operator
   annotations:
+    service.alpha.openshift.io/serving-cert-secret-name: openshift-cluster-version-serving-cert
```

Comment:
[Docs][1] say we can use `service.beta.openshift.io/serving-cert-secret-name` (not alpha) here.  They also have "For convenience, it is recommended that this be the same as `<service-name>`.", which suggests a value of `cluster-version-operator` to match the original Service name or `metrics` to match your rename.


[1]: https://github.com/openshift/openshift-docs/blob/enterprise-4.4/modules/customize-certificates-add-service-serving.adoc

Reply:
This was fixed and spun out to #366.